src/test/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImplTest.java

```diff
@@ -47,6 +48,15 @@ public void setup() {
         return dockerBuildImageProvider;
     }
 
+    private TaskProvider<DockerBuildImage> createBuildDockerImageAlternative(Project project) {
+        TaskProvider<DockerBuildImage> dockerBuildImageProvider = project.getTasks()
```

Comment:
Only extract this to a function if you are going to use it multiple times, else just place the task creation logic inside the test method.